plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -480,6 +487,120 @@ def rpm_search_dicts(self):
                 ret.append(unit_key)
         return ret
 
+    @staticmethod
+    def _check_packages(existing_packages, new_packages):
+        """
+        Check if the new packages are the same as the existing ones.
+
+        :param existing_packages: list of packages presented in the existing erratum
+        :type  existing_packages: list of dicts
+
+        :param new_packages: list of packages presented in the new erratum
+        :type  new_packages: list of dicts
+
+        :return: True, if the lists of packages are equal
+        :rtype: bool
+        """
+        if len(existing_packages) == len(new_packages):
+            existing_packages.sort(key=itemgetter('filename'))
+            new_packages.sort(key=itemgetter('filename'))
+            return all(x == y for x, y in zip(existing_packages, new_packages))
```

Comment:
@seandst, 
In Python 3 `sorted` will fail without a `key`.
But I agree, I can simplify the return expression. Thanks for the hint!
And the length comparison still makes sense... not to sort these lists of dictionaries if it is not needed. 
What do you think?